config/app/server-dev.conf

```diff
@@ -0,0 +1,42 @@
+# server {
```

Comment:
Do we need this file? Might be quite neat to have it around but needs the keys too I guess?

Reply:
It's any old self-signed key, but I can remove this and just keep a note for myself. There's not much to this config anyway.